pb/build_proto/build.pb.go

```diff
@@ -13,9 +13,13 @@ It has these top-level messages:
 */
 package build
 
-import proto "github.com/golang/protobuf/proto"
-import fmt "fmt"
-import math "math"
+import (
+	fmt "fmt"
+
+	proto "github.com/golang/protobuf/proto"
```

Comment:
Did you compile pb.go files using protobuf compiler?

Reply:
For some reason, if I compile the pb.go files using protobuf, then the historian tool is not able to run. However, when the code from the old repo, as long as I get the package path correct, everything is working.